README.md

```diff
@@ -20,18 +20,24 @@ Freshly created code needs to be tested - besides our use of unit tests, linting
 
 ### Vagrant
 
-One can simply set up an execution environment using `vagrant`:
+You can simply set up an execution environment using `vagrant`:
 
 ```bash
-vagrant up --provision
+vagrant up
 ```
 
-At that point, one created a vagrant box, running a [PostgreSQL](https://www.postgresql.org/) database server, [Apache](https://httpd.apache.org/) web server and the [Django](https://www.djangoproject.com/) application. The contents are available on the default port `8000`, which allows one to access the website at `http://localhost:8000`. To create a new superuser for the application, execute `vagrant ssh` in the project directory and trigger the django user management system:
+This will set up a virtual machine and run it. Running this for the first time might take a while.
+
+To connect to it and start the application do:
 
 ```bash
-./manage.py createsuperuser --username=root
+vagrant ssh
+# This will take one inside the virtual machine
```

Comment:
i read this sentence multiple times to understand it. in other places you replaced "one" with "you". that would help a lot here as well :)

Reply:
ah yea I forgot to replace the `one` here :)